src/fqm.ts

```diff
@@ -0,0 +1,18 @@
+import { readFile } from 'fs/promises';
+import { Calculator } from 'fqm-execution';
+import * as path from 'path';
+import { CalculationOptions } from 'fqm-execution/build/types/Calculator';
```

Comment:
```suggestion
import { Calculator, CalculatorTypes } from 'fqm-execution';
import * as path from 'path';
```
The `CalculationOptions` can be accessed by importing `CalculatorTypes` from fqm-execution (see in [README](https://github.com/projecttacoma/fqm-execution#typescript)). This looks like a cleaner approach to me since everything we need from fqm-execution is imported on one line